.circleci/config.yml

```diff
@@ -381,20 +381,22 @@ commands:
           command: |
             echo 'export REDIRECT_URL=$REDIRECT_URL_CONTENT' >> $BASH_ENV
             echo 'export SHOW_DISABLED_INDUSTRIES=true' >> $BASH_ENV
+            echo 'export COGNITO_USER_POOL_ID=$COGNITO_USER_POOL_ID_CONTENT' >> $BASH_ENV
+            echo 'export COGNITO_IDENTITY_POOL_ID=$COGNITO_IDENTITY_POOL_ID_CONTENT' >> $BASH_ENV
+            echo 'export REDIRECT_URL=$REDIRECT_URL_CONTENT' >> $BASH_ENV
 
   set-accessibility-testing-env-vars:
     steps:
       - run:
           name: Set Accessibility Testing Environment Variables
           command: |
+            echo 'export COGNITO_WEB_CLIENT_ID=$COGNITO_WEB_CLIENT_ID_ACCESSIBILITY_TESTING' >> $BASH_ENV
+            echo 'export COGNITO_USER_POOL_ID=$COGNITO_USER_POOL_ID_ACCESSIBILITY_TESTING' >> $BASH_ENV
+            echo 'export COGNITO_IDENTITY_POOL_ID=$COGNITO_IDENTITY_POOL_ID_ACCESSIBILITY_TESTING' >> $BASH_ENV
```

Comment:
[dust] This is more a question for @jphechter, I suppose, but I don't imagine that we would ever want a different `COGNITO_IDENTITY_POOL_ID` (and other Cognitos) for testing and content that is distinct from dev, no?

Reply:
I'd need to sign into AWS to look where the lines of the service end... but don't we need this if we're creating a new connection to myNJ? That'd be the only reason. I think right now they're actually all the same. I'm cool with this if it's a preemptive step.